DEVELOPMENT.md

```diff
@@ -108,7 +108,8 @@ make CONTAINER_ENGINE=podman …
 
 #### Before filing a pull request
 
-Run `docker-compose exec rack rake` while a docker instance is running to run rubocop (to ensure your changes meet the project's code style guidelines) as well as the test suite.
+- Run `git fetch && git diff-tree -r --no-commit-id --name-only master@\{u\} HEAD | xargs ls -1 2>/dev/null | grep '\.rb$' | xargs bundle exec rubocop --force-exclusion -a` to fix formatting errors (after the files are committed, see [StackOverflow](https://stackoverflow.com/questions/32553877/how-to-run-rubocop-only-on-the-changed-files-in-a-pull-request/64272049#64272049))
```

Comment:
It seems a bit confusing to refer to this specific usecase before we refer to linting in general

Could we first add something like
"
`docker-compose exec rack rubocop -a` as a way to run code linting without running the test suite (this is much faster)
"

and then refer to this new note regarding running rubocop on specific files?